roles/lib_openshift/library/oadm_manage_node.py

```diff
@@ -761,6 +762,32 @@ def __init__(self,
         self.verbose = verbose
         self.kubeconfig = Utils.create_tmpfile_copy(kubeconfig)
         self.all_namespaces = all_namespaces
+        self.oc_binary = self._locate_oc_binary()
+
+    @staticmethod
+    def _locate_oc_binary():
+        ''' Find and return oc binary file '''
+        # https://github.com/openshift/openshift-ansible/issues/3410
+        # oc can be in /usr/local/bin in some cases, but that may not
+        # be in $PATH due to ansible/sudo
+        paths = os.environ.get("PATH", os.defpath).split(os.pathsep)
+        extra_paths = ['/usr/local/bin', os.path.expanduser('~/bin')]
```

Comment:
No change needed. Just FYI: this could be compressed to:

```python
paths = list(set(os.environ.get(
    "PATH", os.defpath).split(os.pathsep) + ['/usr/local/bin', os.path.expanduser('~/bin')]))
```

or (for easier reading):

```python
paths = os.environ.get("PATH", os.defpath).split(os.pathsep) + ['/usr/local/bin', os.path.expanduser('~/bin')]
paths = list(set(paths))
```